core/background/services/lastfm.js

```diff
@@ -71,6 +69,10 @@ define([
 				// set token and reset session so we will grab a new one
 				data.sessionID = null;
 				data.token = xml.find('token').text();
+
+				var response = http_request.responseText;
+				console.log('getToken response: %s', response.replace(data.token, data.token.substr(5)));
```

Comment:
In which case would it be better to use to use data.token as it's already parsed?